src/coreclr/vm/comtoclrcall.cpp

```diff
@@ -505,6 +505,13 @@ extern "C" UINT64 __stdcall COMToCLRWorker(Thread *pThread, ComMethodFrame* pFra
         }
     }
 
+    if (pThread->PreemptiveGCDisabled())
+    {
+        EEPOLICY_HANDLE_FATAL_ERROR_WITH_MESSAGE(
+            COR_E_FAILFAST,
+            W("Thread invoking a COM method from non-preemptive GC state. The transition to preemptive mode was missed."));
```

Comment:
```suggestion
            W("Invalid Program: attempted to call a COM method from managed code."));
```
I would use the same wording as `ReversePInvokeBadTransition`